### ISSUE TITLE
Omnidoc fails on JDK11

### ISSUE THREAD

**Issue by ignasi35** [playframework/omnidoc#106]
Running `sbt +publish` for omnidoc using JDK11 ends successfully, even though the output is incomplete and some internal steps actually fail.

See for [example](https://travis-ci.com/playframework/omnidoc/jobs/267569846#L784):

```
[warn] javadoc: error - No source files for package play
[warn] 1 error.
[warn] javadoc exited with exit code 2
[info] Omnidoc Java API documentation successful.
```

I know that at least the `omnidoc/javadoc` folder is empty. But I can't be sure about other expected outputs. There seems to be generated data in both `omnidoc/scaladoc` and `omnidoc/playdoc` but I didn't check if the output is complete.

**Comment by dwijnand** [comment]
This might've been fixed in sbt 1.3, so perhaps worth checking after upgrading.

**Comment by ignasi35** [comment]
> This might've been fixed in sbt 1.3, so perhaps worth checking after upgrading.

LOL #71 and #103 🤦🏼‍♂️

**Comment by mkurz** [comment]
Fixed by #110, also Travis tests against Java 11 meanwhile and looks good.
